<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnumerationToolWidget</class>
 <widget class="QWidget" name="EnumerationToolWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>96</width>
    <height>32</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="schrodinger::sketcher::ModularToolButton" name="rgroup_btn">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">group</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="attachment_point_btn">
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Attachment Point</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../sketcher.qrc">
       <normaloff>:/icons/enumeration_attachment_point.svg</normaloff>
       <disabledoff>:/icons/enumeration_attachment_point_dis.svg</disabledoff>:/icons/enumeration_attachment_point.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">group</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="schrodinger::sketcher::ModularToolButton" name="reaction_btn">
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">group</string>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>schrodinger::sketcher::ModularToolButton</class>
   <extends>QToolButton</extends>
   <header>schrodinger/sketcher/widget/modular_tool_button.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../sketcher.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="group"/>
 </buttongroups>
</ui>
